TYPO3.Neos.NodeTypes/Resources/Private/Templates/NodeTypes/Partials/Image.html

```diff
@@ -5,12 +5,10 @@
 		<f:then>
 			<f:if condition="{link}">
 				<f:then>
-					<a href="{link}"><media:image asset="{image}" alt="{alternativeText}" title="{title}" maximumWidth="{maximumWidth}" maximumHeight="{maximumHeight}" allowCropping="{allowCropping
-}" allowUpScaling="{allowUpScaling}" /></a>
+					<a href="{link}"><media:image asset="{image}" alt="{alternativeText}" title="{title}" width="{width}" maximumWidth="{maximumWidth}" height="{height}" maximumHeight="{maximumHeight}" allowUpScaling="{allowUpScaling}" /></a>
```

Comment:
@dfeyer, but why did you remove `allowCropping` from here? Cropping doesn't work without it, obviously.